packages/sku/entry/libraryRender/index.js

```diff
@@ -1,14 +1,17 @@
 import render from '__sku_alias__renderEntry';
 
 const libraryName = SKU_LIBRARY_NAME;
+const libraryFile = SKU_LIBRARY_FILE;
 const publicPath = __SKU_PUBLIC_PATH__;
```

Comment:
Kinda weird that we have both `SKU_BAR` and `__SKU_BAR__` names. What do you think about normalising to `__SKU_BAR__`?